engine/runtime-integration-tests/src/test/java/org/enso/compiler/dump/test/DocsGenerateTest.java

```diff
@@ -13,49 +13,28 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Set;
-import org.enso.compiler.Compiler;
 import org.enso.compiler.core.IR;
 import org.enso.compiler.core.ir.Module;
 import org.enso.compiler.core.ir.module.scope.Definition;
 import org.enso.compiler.core.ir.module.scope.definition.Method;
 import org.enso.compiler.docs.DocsGenerate;
 import org.enso.compiler.docs.DocsVisit;
 import org.enso.editions.LibraryName;
-import org.enso.interpreter.runtime.EnsoContext;
 import org.enso.pkg.QualifiedName;
+import org.enso.test.utils.ContextRule;
 import org.enso.test.utils.ContextUtils;
 import org.enso.test.utils.ProjectUtils;
 import org.enso.test.utils.SourceModule;
-import org.graalvm.polyglot.Context;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
 public class DocsGenerateTest {
   @ClassRule public static final TemporaryFolder TEMP = new TemporaryFolder();
```

Comment:
FYI: No repeated usage of "rule" in the `TemporaryFolder` case...

Reply:
Named as `ContextUtils` in 952b6752b2254a65fe63107c919021824066cb6c